torchaudio/csrc/decoder/bindings/_decoder.cpp

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
I think `_decoder.cpp`, `_dictionary.cpp` and `pybind.cpp` can be combined and placed directly at `torchaudio/csrc/decoder/*`. it looks strange that `pybind.cpp` incldues two other cpp files.